### ISSUE TITLE
Add new lifter not working

### ISSUE THREAD

**Issue by pavelrahman619** [meethed/simpleLifter#35]
**Describe the bug**
The Add New lifter in Tech Desk page does not work.

**To Reproduce**
1. Go to 'Simple Lifter > Competition Management'
2. Click on 'Tech Desk'
2. Click on 'Add New Lifter'

**Expected behavior**
I should be able to add a new lifter

**Screenshots**
![image](https://github.com/user-attachments/assets/8e3fac6a-97aa-47ef-8ae3-057cd36e9d2f)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser Chrome
 - Version 28.0.6613.138 (Official Build) (64-bit)

**Smartphone (please complete the following information):**

**Additional context**

**Comment by meethed** [comment]
Hey mate I'm assuming you meant for the testing competition? 

It's fixed there for now. Can you confirm it was only that competition? The root cause - deleting the d1s1p1 (the only mandatory session) when there was only one session assigned to the competition broke the loader as it was looking for the default values. I'll fix that tonight.

**Comment by pavelrahman619** [comment]
Yes it's still not working in the live site link: http://cotf.zapto.org/simpleLifter2.php. I added a test competition called BPA Test 1 but I am not able to Add New Lifters

**Comment by meethed** [comment]
hi pavel - i've just checked and you somehow had the same problem where the first session had been deleted. i have fixed this for your competition where you should now be able to add lifters again. i dont' know your access code so can't log in exactly as you would to check, however i did replicate it and verify on the "Test password is 2024" competition.

i've now put a process in place to prevent that from happening and have created a new competition to verify that the add lifter button works. for a busier competition, i would still recommend that you use the comp setup menu, rather than the add lifter button.

Please let me know if this has helped

**Comment by pavelrahman619** [comment]
I verified on the "Test password is 2024" competition in the live site and its working. Can you provide a sample database backup file and push to the github repository so I can test it locally?

**Comment by meethed** [comment]
hey mate i can do that, just so im' clear with your question though:

Are you self-hosting using code cloned/pulled from here?

**Comment by pavelrahman619** [comment]
Currently I am just testing in my local host. I created a sample database using the DATABASE.md file you have shared in the repository but there are missing columns in it as the project is not running properly. Most likely there's been new columns added so it needs to be updated.

**Comment by meethed** [comment]
Hi Pavel, I created a new version of database.md with an SQL dump. It should work now.

It was very outdated, I had kinda given up on updating some of those documents I'm really sorry...

**Comment by pavelrahman619** [comment]
I tried uploading the SQL query given in the database.md to my phpmyadmin database but faced issues with these lines:

`
/*!40101 SET character_set_client = @saved_cs_client */;
/*!40103 SET TIME_ZONE=@OLD_TIME_ZONE */;

/*!40101 SET SQL_MODE=@OLD_SQL_MODE */;
/*!40014 SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS */;
/*!40014 SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS */;
/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
`

It is a much better practice to use the export feature in phpMyAdmin to export a .sql file of the database. I removed the code, ran the query, and it works so far. However, I am now facing a new issue with the latest update that calls a .config.inc file. I will create a separate issue for that. As an aspiring powerlifter and a Laravel web developer by profession, I am open to contributing to this project at any time with my professional expertise.
